blog/views.py

```diff
@@ -0,0 +1,44 @@
+from django.views.generic import ListView, DetailView, CreateView, DeleteView, UpdateView
+from django.urls import reverse_lazy, reverse
+from blog.models import Blog
+
+
+class BlogListView(ListView):
+    model = Blog
+    template_name = 'blog/blog_list.html'
+    context_object_name = 'blogs'
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        return queryset.filter(publication_attribute="True")
```

Comment:
кавычки не нужны - это булево поле
```suggestion
        return queryset.filter(publication_attribute=True)
```